src/outputs/outputs_main.py

```diff
@@ -170,6 +171,20 @@ def run_outputs(
         )
         OutputMainLogger.info("Finished  Intram by ITL1 output.")
 
+    # Running Intram UK by ITL 1 and 2
+    if config["global"]["output_intram_uk_itl2"]:
+        OutputMainLogger.info("Starting  Intram UK by ITL 1 and 2 output...")
+        output_intram_uk_itl_1_2(
+            outputs_df,
+            ni_full_responses,
+            config,
+            write_csv,
+            run_id,
+            postcode_mapper,
+            itl_mapper,
+        )
+        OutputMainLogger.info("Finished  Intram UK by ITL 1 and 2 output.")
+
```

Comment:
The totals are matching each other, and I tried to check against GB SAS + NI SAS and they were't tallying up.  Just had a quick look at the after the merge, and there are some postcodes not being mapped (e.g. GU16 4FF, EC2V 2JE, OX26 5HR) so possibly being dropped
![image](https://github.com/ONSdigital/research-and-development/assets/124355509/296eac06-ab56-4f61-9d8b-c2bc7bd915c4)

Reply:
Good check. The sample on R: is not constructed, so some wrong codes are not mapped. This needs to be checked in DAP where  all postcodes should map to reg_code and itl